RELEASE.md

```diff
@@ -1,13 +1,11 @@
 # How to Release
 
-Releasing the project requires these steps:
+This project uses [semantic-release](https://github.com/semantic-release/semantic-release) to automatically release new versions when commits are merged to the master branch.  The semantic-release process is triggered as part of the [Travis build](https://travis-ci.com/cerner/cucumber-forge-report-generator).
 
-0. Set the version number in the code
-1. BUILD-COMMAND
-2. Use a GitHub [project release][github-release-url] to release the project and tag (be sure it follows [semver][semantic-versioning])
-3. PACKAGE-COMMAND and/or RELEASE-COMMAND
-4. Update `master` to a new minor version
+To ensure that releases are triggered properly, the following standards should be applied to commit messages for the master branch:
 
-[project-url]: https://github.com/cerner/cucumber-forge-report-generator/
-[semantic-versioning]: http://semver.org/
-[github-release-url]: https://help.github.com/articles/creating-releases/
+| Commit message                                                                                                                                                                                   | Release type               |
+|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------|
```

Comment:
could we ditch some of the lines since they aren't needed in markdown?

Reply:
Yeah. sorry about that. Just lifted this directly from the semantic-release README and did not really thing to clean it up.